themes/default/content/blog/cloud-engineering-fuels-startup-innovation/index.md

```diff
@@ -0,0 +1,94 @@
+ 
+Within the [cloud engineering]({{< relref "/cloud-engineering" >}}) community, we see several common patterns for harnessing the modern cloud. Some startups have teams of full stack developers who need to rapidly and safely deploy cloud infrastructure and applications at high velocity. Others might have a few infrastructure or platform engineers who need to enable other developers to easily use cloud infrastructure on a self-serve basis. Many of these teams started off using domain-specific languages (DSLs) to manage infrastructure as code and quickly found that these languages were the limiting factor in achieving faster velocity. DSLs are cumbersome to use and don’t support the logic and expressiveness needed to build and manage modern architectures that are more complex in nature. DSL’s are also a barrier to entry to most developers. 
+ 
+In contrast, cloud engineering makes cloud infrastructure more accessible and easier to use for everyone with tried and true software engineering practices and tools. It embraces the reality that [all software is now cloud software]({{< relref "/blog/future-of-cloud-engineering/" >}}). A core practice is using a commonly used language like TypeScript, Python, Go, or C#. That’s one reason why more and more startups are choosing Pulumi to help them achieve competitive advantage through cloud infrastructure. It also doesn’t hurt that Pulumi was [rated the “Top Paying Technology” by developers]({{https://insights.stackoverflow.com/survey/2021#top-paying-technologies-other-tools}}) in 2021. 
+
+Cloud engineering is a central character in the next chapter of how the modern cloud fuels startup innovation. Read on to see three mini-stories of cloud engineering in action with startups. 
+
+<!-- ## Panther Labs -->
+Panther Labs helps modern security teams build world-class detection and response pipelines using code and automation, developer-friendly workflows, and big data primitives. Its Platform Team is responsible for a large, complex serverless architecture on AWS. Because of the limitations of its legacy Infrastructure-as-Code (IaC) tool, the team was unable to manage and scale its cloud infrastructure with the speed and automation that the company needed to support its fast-growing business. After comparing different alternatives, Panther Labs decided to migrate to the Pulumi Cloud Engineering Platform. Pulumi increased the company’s deployment speeds by up to 10X, reduced the size of its infrastructure codebase by >50%, and enabled its developers to adopt cloud engineering best practices to deliver its cloud applications faster and more reliably.
+
+### Results
```

Comment:
Our linter requires a blank line after every heading. Generally, static site generators could trip up over the lack of space around a heading, so many linters for these kinds of systems will call those out.
```suggestion
### Results

```